mlops/parallelm/mlops/ml_metrics_stat/classification/classification_report.py

```diff
@@ -0,0 +1,51 @@
+from parallelm.mlops.metrics_constants import ClassificationMetrics
+from parallelm.mlops.mlops_exception import MLOpsStatisticsException
+from parallelm.mlops.stats.table import Table
+
+
+class ClassificationReport(object):
+    """
+    Responsibility of this class is basically creating table stat object out of classification report string.
+    """
+
+    @staticmethod
+    def get_mlops_classification_report_table_stat_object(cr):
+        """
+        Method will create MLOps table value stat object from classification report string.
+        It is not recommended to access this method without understanding single value data structure that it is returning.
+        :param cr: classification report string.
+        :return: Table Value stat object which has classification report embedded inside
+        """
+        table_object = None
+        if cr is not None:
+            try:
+                array_report = list()
+                for row in cr.split("\n"):
```

Comment:
Does it make sense to check that cr is String instance

Reply:
i tried.. i actually had that check as well. but .. it can be of str or unicode. depends on sklearn version that user have! 
thus i had to remove it.

Reply:
i added check to beautify patch: https://github.com/mlpiper/mlpiper/pull/93